### PR TITLE
Symbian: update configs for 2.5.1 release

### PR DESCRIPTION
**Pull request by fedor4ever**

### CODE REVIEW

**Comment by Die4Ever** [comment]
lots of these files are part of .gitignore, something went wrong here

File: backends/platform/psp/README.PSP [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ScummVM-PSP 2.5.1pre README
+ScummVM-PSP 2.5.0 README
```
Comment (Die4Ever) [inline backends/platform/psp/README.PSP:1]: shouldn't this say 2.5.1?

**Comment by fedor4ever** [comment]
Well, I wish to push changes for several Symbian config files but something wrong happened.

How resolve this? Any help welcome. I have no idea.

**Comment by sev-** [comment]
This is all wrong: it contains unrelated commits and the commits do not conform to our commit guidelines.

**Pull request closed without merging**